cartridge/twophase.lua

```diff
@@ -293,6 +294,33 @@ local function patch_clusterwide(patch)
     return ok, err
 end
 
+function _G.cartridge_get_schema()
+    if confapplier.get_readonly() == nil then
+        return nil, GetSchemaError:new(
+            "Cluster isn't bootstaraped yet"
+        )
+    end
+    local schema_yml = confapplier.get_readonly('schema.yml')
+
+    if schema_yml == nil then
+        return {spaces = {}}
+    else
+        return yaml.decode(schema_yml)
+    end
+end
+
+function _G.cartridge_set_schema(schema)
```

Comment:
this should take YAML

Reply:
The intention was to provide API useful for connectors. They don't care about formatting.